issue_event_processing.ipynb

```diff
@@ -0,0 +1,458 @@
+   "metadata": {},
+   "source": [
+    "# Introduction\n",
+    "Process mining helps uncover insights from data that represent different events in a process. It is valuable for understanding workflows, finding inefficiencies, and optimizing development processes. Using Kaiaulu  for process mining enables this discovery for Github Events (e.g. creating issues, push events, deleting branches, etc). Allowing users to visualize and analyze event data effectively. This notebook demonstrates how to use Kaiaulu to download and parse Github Event data. As well how to interact with the data to leverage process discovery for deeper analysis. This [video](https://www.youtube.com/watch?v=XLHtvt36g6U&t=1181s) provides an overview of process mining concepts and its applications and can be used as a reference point to begin. This notebook heavily relies on the python package [pm4py](https://github.com/process-intelligence-solutions/pm4py) to create event mappings.\n",
+    "\n",
+    "This notebook walks through multiple different models and algorithms for modeling the process. It is reccomended you start with a small number of event issues and gradually build multiple versions to have a deeper understanding of what events are changed the process model. At the bottom of the notebook is an examples section that shows what the models can generate. "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Requirements\n",
+    "\n",
+    "-   Gihub Access Token \n",
+    "-   [Kaiaulu](https://github.com/sailuh/kaiaulu) R package to download data via CLI\n",
+    "-   Python environment with [pm4py](https://github.com/process-intelligence-solutions/pm4py) installed\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Python Imports"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 137,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import pandas\n",
+    "import os\n",
+    "from datetime import datetime\n",
+    "import pm4py\n",
+    "import subprocess\n",
+    "\n",
+    "from pm4py.algo.discovery.dfg import algorithm as dfg_discovery\n",
+    "from pm4py.algo.discovery.heuristics import algorithm as heuristics_miner"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Download and Parse data with Kaiaulu ghevents.R (CLI)\n",
+    "\n",
+    "#### Set Up and Execute Download Command\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Save and set the current working directory to Kaiaulu\n",
+    "cwd = os.getcwd()\n",
+    "os.chdir(os.path.expanduser(\"~/Desktop/Kaiaulu/Working_issues/kaiaulu/\"))\n",
+    "\n",
+    "# To download use the download command specifing the <config_file> <github_token>\n",
+    "\n",
+    "command = f\"Rscript exec/ghevents.R download conf/kaiaulu.yml --token_path=~/.ssh/github_token\"\n",
```

Comment:
This is not a good path. It assumes the user is running inside the Kaiaulu project the exec. Rather, the reasonable assumption is that the user downloaded your code (process_mining) and wants to run the exec from Kaiaulu. So the current working directory should assume the user is at process_mining folder, goes one level above, then access the kaiaulu folder. 

What I recommend you do is to explain the expected folder organization of the projects before throwing paths left and right. Assume the following:

kaiaulu/kaiaulu/exec/ghevents.R
kaiaulu/rawdata/
process_mining/notebooks/issue_event_processing.ipynb